libcxx/test/support/filesystem_test_helper.h

```diff
@@ -582,7 +582,11 @@ struct ExceptionChecker {
     assert(ErrorIsImp(Err.code(), {expected_err}));
```

Comment:
Not attached to this file:

In the PR description, you mention:

> This allows code like:
> ```
> error_code(GetLastError(), system_category()) == errc::invalid_argument
> ```
> to work as expected -- as it does with MSVC STL.

I don't think I saw any test added to validate that this worked as intended. Did I miss that or should we add a test?